src/Open3D/GUI/Color.cpp

```diff
@@ -50,9 +50,16 @@ float Color::GetGreen() const { return rgba_[1]; }
 float Color::GetBlue() const { return rgba_[2]; }
 float Color::GetAlpha() const { return rgba_[3]; }
 
+void Color::SetRGB(const float r, const float g, const float b) {
+    rgba_[0] = r;
+    rgba_[1] = g;
+    rgba_[2] = b;
+}
+
 const float* Color::GetPointer() const { return rgba_; }
+float* Color::GetMutablePointer() { return rgba_; }
 
-Color Color::Lightened(float amount) {
+Color Color::Lightened(const float amount) {
```

Comment:
It is const not to prevent writing to source variable, from which we copied amount) I have rule of thumb to set all function parameters to const to not modify them by occasion in functions body.